### ISSUE TITLE
Next and previous page buttons not working, add command alternative

### ISSUE THREAD

**Issue by NEXUS2345** [prism/Prism#127]
The current version appears to have a broken page forward and back buttons in the chat printout. In addition to getting this fixed, could the old commands for moving pages be readded from the previous versions?

**Comment by LXGaming** [comment]
Try the latest build from here: http://helion3.com:8080/job/Prism/ the buttons work fine (They are only clickable if there are more pages available).

**Comment by NEXUS2345** [comment]
Yeah, that has worked for me on a vanilla Sponge server. I think the issue might be a mod that is modifying the chat window in some modpacks. I notice there is in fact a command, /page next and /page previous, could this be added to the documentation?

**Comment by phit** [comment]
`/page` is a sponge command
you can also do `/page 3` to go to the third page etc, this works with all paginated text pages in sponge e.g. `/help`
